Press Badge

Rebruiza opened the pull request:
:wave:

Drone made smaller as it is a fluff item and offers no gameplay advantages.

comma commented on code/game/objects/items/devices/tvcamera.dm:
```diff
@@ -1,11 +1,11 @@
 	icon_state = "camcorder"
 	item_state = "camcorder"
-	w_class = ITEM_SIZE_LARGE
+	w_class = ITEM_SIZE_NORMAL
```
It does in fact offer a gameplay advantage of mobile camera for security etc to look through.

comma commented:
Overall, I very don't like the 'grammar' changes.
Descriptions of meaningless detail like 'it hovers' worded in dryest way possible, removing one of few ingame mentions of things existing in lore.

BlueNexus commented:
Please don't remove lore just because you don't like it.

comma commented on code/game/objects/items/devices/tvcamera.dm:
```diff
@@ -1,11 +1,11 @@
-	w_class = ITEM_SIZE_LARGE
+	w_class = ITEM_SIZE_NORMAL
 	slot_flags = SLOT_BELT
-	var/channel = "NSS Exodus News Feed"
```
Doesn't this result in broken camera with null tag? Just change to generic 'News Feed'

BlueNexus commented on code/game/objects/items/devices/tvcamera.dm:
```diff
@@ -1,11 +1,11 @@
 /obj/item/device/tvcamera
 	name = "press camera drone"
-	desc = "A Ward-Takahashi EyeBuddy media streaming hovercam. Weapon of choice for war correspondents and reality show cameramen."
+	desc = "A livestreaming press camera drone. It appears to be following the controller itself, and stabilizes at around head height. It does not appear to have any internal memory storage."
```
This has nothing to do with grammar, this is just the removal of lore. Why'd you change this?

comma commented on code/game/objects/items/devices/tvcamera.dm:
```diff
@@ -1,11 +1,11 @@
 /obj/item/device/tvcamera
 	name = "press camera drone"
-	desc = "A Ward-Takahashi EyeBuddy media streaming hovercam. Weapon of choice for war correspondents and reality show cameramen."
+	desc = "A livestreaming press camera drone. It appears to be following the controller itself, and stabilizes at around head height. It does not appear to have any internal memory storage."
```
This is not 'grammar' fix, you're changing whole thing, to a boring ass 'description' that describes things you can see looking at the sprite. Please don't.

Rebruiza replied:
This was done with feedback from a journalist player.

comma replied:
'please remove mentions of corporations and add menial details instead'?

comma commented on code/modules/clothing/under/accessories/badges.dm:
```diff
@@ -6,18 +6,13 @@
 	name = "detective's badge"
-	desc = "A simple badge, made from gold."
-	icon_state = "goldbadge"
+	desc = "A detective's badge, made from gold."
```
Ingame it would look like
```
This a detective's badge.
A detective's badge. ...
```
And again, what's the wrong 'grammar' here you 'fixed'?

Rebruiza commented:
kk im gonna fix the complaints

comma commented on maps/torch/job/outfits.dm:
```diff
@@ -675,7 +675,8 @@ Keeping them simple for now, just spawning with basic EC uniforms, and pretty mu
 	name = OUTFIT_JOB_NAME("Journalist - Torch")
-	backpack_contents = list(/obj/item/device/tvcamera = 1)
+	backpack_contents = list(/obj/item/device/tvcamera = 1,
+	/obj/item/clothing/accessory/badge/press = 1,
```
Missing ) here

BlueNexus commented on code/modules/clothing/under/accessories/badges.dm:
```diff
@@ -124,4 +124,10 @@
+	
+/obj/item/clothing/accessory/badge/press
+    name = "press badge"
+    desc = "A press badge."
```
"press badge"
"a press badge" 
At least try.

Rebruiza replied:
i'll slap u

ghost commented:
:clap: travis :clap: please :clap:

comma commented:
>4 incorrect indentations (expecting exactly 0)

Somewhere spaces are used instead of tabs

>Could not parse property 'dirs': invalid literal for int() with base 10: '2"' etc

Check your icons, see if there are ' or , in iconstates names there

PsiOmegaDelta commented:
`Could not parse property 'dirs': invalid literal for int() with base 10: '2"' etc` doesn't result in Travis failures. It's just a warning.

Commit:
Adds several things


Makes the default channel more generic


Grammar


Fixes


Grammar


Angry People Fix?


Updates outfits.dm


Updates tvcamera.dm


Updates badges.dm


Icon Fixes

ghost commented:
Accidentally merged in the first place with some work in progress icons. They have been removed.

ghost commented:
@comma Why isn't Travis completing CI?

comma commented:
Because there's conflicts, rebase to dev and resolve them - your best bet would be to save your new icons somewhere, then accept upstream/dev version of dmi, and readding them again manually.

ghost commented:
something died so i'm remaking the PR
